spec/tile_bag_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe "class TileBag" do
+  before do
+   @tile_bag = Scrabble::TileBag.new
+  end
+
+  describe "initialize" do
+    it "sets up the instance with a collection of all default tiles" do
+      @tile_bag.must_be_instance_of Scrabble::TileBag
+    end
+  end
+
+  describe "draw tiles" do
+    it "returns nil if input is more than tilebag" do
+      @tile_bag.draw_tiles(100).must_be_nil true
+    end
+
+    it "returns an array of tiles to user" do
+      @tile_bag.draw_tiles(3).must_be_kind_of Array
+    end
+
+    it "returns an empty array of tiles to user" do
+      @tile_bag.draw_tiles(0).must_be_empty Array
+    end
```

Comment:
Good failure cases here! I like that you both attempt to draw too many tiles and to draw 0.

I would also like to see a case where you draw tiles until the last tile has been drawn (this should work).